Add definition for blacklist/whitelist, fix styling of blockquotes

deammer opened the pull request:
This PR addresses #207 by adding a definition for the terms blackist and whitelist.

This is my first contribution to SelfDefined and I welcome all the feedback I can get!

I followed the definition of `master/slave` in that I created one file for both terms, but maybe it would be better to break this up into 2 definitions?

Thank you in advance to the editor ❤️

Commit:
Add definition for blacklist/whitelist

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,34 @@
+---
+title: Blacklist Whitelist
```
```suggestion
title: blacklist whitelist
```

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,34 @@
+  level: avoid
+  text: racist term
+defined: true
+exceprt: terms used in many contexts to represent a list of entities or people who are to be allowed or denied access to a service, privilege, or recognition. To "blacklist" is "to exclude", to "whitelist" means "to include".
```
```suggestion
except: umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include".
```

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,34 @@
+  - 'whitelist (verb): allow, trust, include'
+reading:
+  - text: Basecamp stops using black/whitelist (tweet)
+    href: https://twitter.com/dhh/status/1032050325513940992
```
I'd suggest replacing this tweet with https://www.techspot.com/news/85080-uk-cyber-security-agency-stop-using-racist-blacklist.html

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,34 @@
+
+## Issue
+
+Using a color to denote whether an entity is included or excluded from a group draws parallels to the idea that Black and White people have respectively been denied and allowed privileges based on the color of their skin. Similarly to the concept of [master/slave](/definitions/master-slave), blacklist/whitelist is used casually in the realm of software development, which reinforces the exclusion  ...
+
```
Under this area, I'd love a bit more finessing and context added with the historical context and origins of the term:

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/ offers some tactical insights about where the terms first began and the context of slavery.

I suggest that we parse the issues into two main buckets of issues:
- Black/white as a good/bad dichotomy
- Black/white admittance and denial (Jim Crow)

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,34 @@
+exceprt: terms used in many contexts to represent a list of entities or people who are to be allowed or denied access to a service, privilege, or recognition. To "blacklist" is "to exclude", to "whitelist" means "to include".
+speech: noun; verb
+alt_words:
+  - 'blacklist (noun): blocklist, denylist, excludelist'
+  - 'whitelist (noun): allowlist, passlist, trustlist, safelist'
+  - 'blacklist (verb): block, deny, exclude'
+  - 'whitelist (verb): allow, trust, include'
```
Here you can take advantage of some nestted terms here to keep this a bit neater:

You can see the [dude definition](https://www.selfdefined.app/definitions/dude/) as an example of how this might work.

deammer replied:
Oh this is awesome!

tatianamac reviewed (changes requested):
@deammer Thank you for submitting this PR for this definition! It's a really solid start. I provided a few suggestions that I think will enrichen the definition.

tatianamac commented:
@deammer Hi! Do you still have the capacity/desire to finish out this definition? No worries on my side either way, just hoping to update statuses for open PRs. Thank you for all your work on it already!

Commit:
Code review updates

Commit:
Yaml fix

Commit:
Organized alt words and fix blockquote styling

deammer commented on assets/css/structures/_definition-content.scss:
```diff
@@ -57,12 +57,18 @@
 
     & h2 ~ p,
-    & h2 ~ ul {
+    & h2 ~ ul,
+    & h2 ~ blockquote {
       font-size: inherit;
       // border-left: 0.1rem solid lightgrey;
       // padding-left: 1rem;
       grid-column: 2;
     }
+
+    & h2 ~ blockquote {
+      border-left: 0.1rem solid lightgrey;
+      padding-left: 1rem;
+    }
```
I took the liberty of fixing the styling of blockquotes. They currently render in a very slim column (see the [optical allyship definition](https://www.selfdefined.app/definitions/optical-allyship/) for an example), so I styled them like this:

<img width="733" alt="Screen Shot 2020-11-17 at 11 18 05 AM" src="https://user-images.githubusercontent.com/3411183/99437120-e543e600-28c6-11eb-90dc-3ed2696c0d52.png">

Please let me know if you have any feedback or would rather fix this in a different PR.

tatianamac replied:
Oh yikes, thanks for catching and fixing this. This is perfect! Normally I'd ask to split features into a separate PR, but I think it's fine so long as we rename your PR accordingly.

deammer replied:
Renamed the PR 👍

Commit:
Refined the wording

deammer commented:
@tatianamac thank you for the nudge, I appreciate it. I've fixed all the issues you pointed out, and I attempted to improve the copy. I'd love some feedback on it whenever you have some time 🙏

deammer commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,63 @@
+
+To allow or deny access to a service, a privilege, or recognition.
+
+## Issues
```
I'd love some feedback on this section especially

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,63 @@
+    href: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/
+---
+
+To allow or deny access to a service, a privilege, or recognition.
```
```suggestion
colloquially: umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include;" carries anti-Black sentiments
```
I think what you have in the excerpt is honestly more complete. I also added the last phrase because we try to include the issues immediately so that if people only read the top level definition, they get this context.

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,63 @@
+  level: avoid
+  text: racist term
+defined: true
+excerpt: 'umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include".'
```
```suggestion
excerpt: 'umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include;" carries anti-Black sentiments'
```

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,63 @@
+
+<!-- Black/white admittance and denial (Jim Crow) -->
+
+The first problem is the admittance and denial of people based on their race and skin color. The term are rooted in racism: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
```
```suggestion
The first problem is the admittance and denial of people based on their race and skin color. The terms are rooted in racist history: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
```
I think here we need to be mindful of breaking down this connection a bit more. According to the research I did, `blacklist` came from the 1600s and "black books." So, while _etymologically_ the origins predate American slavery, it's that the terms black and white in *modern* usage carry a racial meaning. By using these terms, we're reinforcing the idea that black is bad and white is good, and therefore, reinforcing the systemic and infrastructural racism in a small and powerful way.

Mostly I want to ensure that "well actually" people don't "get us" on a technicality (and also I want to ensure we're promoting as much historical accuracy as possible, while still reinforcing the idea that there are better terms.)

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,63 @@
+
+The first problem is the admittance and denial of people based on their race and skin color. The term are rooted in racism: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
+
+Using "black" and "white" to determine whether an entity is included or excluded from a group is correlated to the racist idea that Black and White people have been respectively denied and allowed privileges based on the color of their skin. Similarly to the concept of [master/slave](/definitions/master-slave), blacklist/whitelist is used casually in the realm of software development, which reinf ...
```
```suggestion
Using "black" and "white" to determine whether an entity is excluded or included, respectively, from a group supports the systemic racism that denies Black people entry while allowing white people entry. Similarly to the concept of [master/slave](/definitions/master-slave), blacklist/whitelist is used abundantly and casually in the realm of software development, which can reinforce the systemic exclusion of Black people from the industry.
```

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,63 @@
+<!-- Black/white as a good/bad dichotomy -->
+
+The second issue with these terms is the implication that black is bad and white is good. A "blacklist" often refers to a list of undesirable things (spammy phone numbers, undesired guests, etc.) whereas a "whitelist" refers to a list of virtuous things (important guests, emails we want to receive, etc.) In 1610, the _Douglas Harper Etymology Dictionary_ defined the term "blacklist" as a "list of ...
+
```
```suggestion
While the origins predate the transatlantic slave trade, the concept of a blacklist as eliciting "suspicion" reinforce the systemic racism that causes disproportionately [higher incarceration and incrimination of Black people](https://www.washingtonpost.com/graphics/2020/opinions/systemic-racism-police-evidence-criminal-justice-system/) for equivalent crimes to non-Black people. 
```
Adding a clause such as this will help to prevent the "gotchas" I shared above.

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,63 @@
+
+## Usage Tip
+
+Seek to change any existing "blacklist/whitelist" terminology to one of the suggested terms, which have the advantage of being more descriptive and less jargony.
```
```suggestion
Be more specific: Not only are "blacklist/whitelist" racist, they tend to be jargony and change from usage to usage as to what it means.
```

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,63 @@
+
+Seek to change any existing "blacklist/whitelist" terminology to one of the suggested terms, which have the advantage of being more descriptive and less jargony.
+
+Even if you may not be negatively impacted by this terminology, it can negatively impact other existing and prospective contributors onto your project. The terminology isn't so unique that it's irreplaceable; the lift tends to be trivial in comparison to the potential impact.
```
```suggestion
Even if you may not be negatively impacted by this terminology, it can negatively impact other existing and prospective contributors onto your project and spaces. The terminology isn't so unique that it's irreplaceable; the lift tends to be trivial in comparison to the potential impact.
```

tatianamac reviewed (changes requested):
Very very close imho! 👍🏽

tatianamac commented:
Thank you for all the changes, @deammer ! I left some more notes to provide even more nuance. Hopefully my notes make sense. :)

deammer committed:
Update summary

Co-authored-by: Tatiana Mac <github@tatianamac.com>

deammer committed:
Update excerpt

Co-authored-by: Tatiana Mac <github@tatianamac.com>

deammer committed:
Apply suggestions from code review

Co-authored-by: Tatiana Mac <github@tatianamac.com>

Commit:
Minor wording improvements

deammer commented:
@tatianamac your notes were very helpful so I've incorporated them, along with some minor tweaks and additional line breaks. I've also renamed the PR as you suggested. Let me know if you'd like any more changes!

tatianamac commented on 11ty/definitions/blacklist-whitelist.md:
```diff
@@ -0,0 +1,65 @@
+
+<!-- Black/white admittance and denial (Jim Crow) -->
+
+The first problem is the admittance and denial of people based on their race and skin color. The terms are rooted in racist history: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
```
```suggestion
The first problem is the admittance and denial of people based on their race and skin color. The terms are rooted in racist history: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
```
I think we'll want to remove/adjust the above section, as to me it feels in direct contradiction with line 53?

deammer replied:
You're right. I've moved things around a little: I kept the reference to the 1610 definition because I think the correlation between the peak of the transatlantic slave trade and the term being coined in a dictionary is important, but I fixed the chronological inconsistency.

tatianamac reviewed (approved):
One more line—thank you for your immense patience! 🙏🏽

tatianamac commented:
Radical. Thank you, @deammer ! I think one line feels out of place/contradicts a later line. once that's resolved and we update the branch, we should be good to go! 🎉🚀

Commit:
Fixed a contradiction about the origin of the terms

deammer commented:
@tatianamac back at you! Thanks for your patience :D

tatianamac reviewed (approved):
🚀

tatianamac commented:
Thank you, @deammer, for your contribution! I hope you'll contribute again soon. 🙏🏽

mlindner commented:
These words have no basis in racism. They long pre-date African slave trade and date from the 1600s.
